packages/model-viewer/src/three-components/ARRenderer.ts

```diff
@@ -515,17 +509,37 @@ export class ARRenderer extends EventDispatcher {
         null;
   }
 
-  public placeModel(hit: Vector3) {
-    this.placementBox!.show = true;
+  public moveToFloor(frame: XRFrame) {
```

Comment:
Maybe `moveToPlane` since there's `this.placeOnWall` in this method

Reply:
Good call; thanks for the drive-by review 😁